docs/_getting-started/index.md

```diff
@@ -44,25 +44,31 @@ You can connect a Geth node to several different networks using the network name
 These include the main Ethereum network, [a private network](getting-started/private-net) you create,
 and three test networks that use different consensus algorithms:
 
--   **Ropsten**: Proof-of-work test network
--   **Rinkeby**: Proof-of-authority test network
--   **Görli**: Proof-of-authority test network
+-   **Ropsten**: Proof-of-work test network
+-   **Rinkeby**: Proof-of-authority test network
+-   **Görli**: Proof-of-authority test network
 
 For this guide, we use the Görli network.
 
+Because of the peer to peer nature of Ethereum the system running Geth has to:
+
+1. Have a publicly accessible IP address (it does not have the be the same as the private IP)
```

Comment:
That's not quite true. As long as one out of two peers has public IP, they can talk. Also, for devnets, both nodes can be on the same internal NATted network, that's fine too.

Reply:
I disabled incoming access to port 30303 and it still works. So the entire paragraph is wrong and I'll delete it.